.github/workflows/pr.yml

```diff
@@ -70,6 +70,15 @@ jobs:
           echo "generated sources are not up to date"
           exit 1
         fi
+    -
+      name: Check deployment files
+      run: |
+        git reset HEAD --hard
```

Comment:
Why do we need it? I see that we need it for check format, since `go get -u` modifies go.mod, but here if there is any change happened - we should fail earlier (the same question for the step above)

Reply:
I'm just following the format we have for other tasks :)

Ideally, CI would fallthrough and report all issues but I understand why it doesn't.

Reply:
Removed these lines